dot/network/service.go

```diff
@@ -290,6 +320,47 @@ func (s *Service) collectNetworkMetrics() {
 	}
 }
 
+func (s *Service) getTotalStreams(inbound bool) int64 {
+	var count int64
+	for _, conn := range s.host.h.Network().Conns() {
+		for _, stream := range conn.GetStreams() {
+			if isInbound(stream) && inbound || !isInbound(stream) && !inbound {
+				count++
+			}
+		}
+	}
+	return count
+}
+
+func (s *Service) getNumStreams(protocolID byte, inbound bool) int64 {
+	np, has := s.notificationsProtocols[protocolID]
+	if !has {
+		return 0
+	}
+
+	var hsData *sync.Map
+	if inbound {
+		hsData = np.inboundHandshakeData
+	} else {
+		hsData = np.outboundHandshakeData
+	}
+
+	var count int64
```

Comment:
nit Define `count` as the returned variable

```suggestion
func (s *Service) getNumStreams(protocolID byte, inbound bool) (count int64) {
	np, has := s.notificationsProtocols[protocolID]
	if !has {
		return 0
	}

	var hsData *sync.Map
	if inbound {
		hsData = np.inboundHandshakeData
	} else {
		hsData = np.outboundHandshakeData
	}

```

Reply:
done